TourGuide_Horde/37_Alterac_Mountains.lua

```diff
@@ -3,12 +3,14 @@ return [[
 F Tarren Mill
 A Prison Break In |N|From Magus Wordeen Voidglare (61.6, 20.9)| |Z|Hillsbrad Foothills| |NODEBUG| |QID|544| |NODEBUG|
 A Stone Tokens |N|From Keeper Bel'varil| |QID|556|
+K Mountain Lion |L|5810|
```

Comment:
Add a note here to include that we are looting a Fresh Carcass and maybe a coord towards the cave we will be heading to. (37,66)

Reply:
Will change.